site/src/pages/docs/setup.mdx

```diff
@@ -86,15 +86,15 @@ You might want to use the following build file as a starting point when you set
         ['armeria',
         'armeria-brave',
         'armeria-grpc',
-        'armeria-jetty',
+        'armeria-jetty9',
         'armeria-kafka',
         'armeria-logback',
         'armeria-retrofit2',
-        'armeria-rxjava',
+        'armeria-rxjava3',
         'armeria-saml',
-        'armeria-thrift',
-        'armeria-tomcat',
-        'armeria-zookeeper'].each {
+        'armeria-thrift0.13',
+        'armeria-tomcat9',
+        'armeria-zookeeper3'].each {
             compile "com.linecorp.armeria:\${it}:${versions['com.linecorp.armeria:armeria-bom']}"
```

Comment:
How about using `api` and `runtimeOnly` below?

Reply:
That a good idea. Let me use `implementation` and `runtimeOnly`